boards/x86/qemu_x86/Kconfig.defconfig

```diff
@@ -9,3 +9,22 @@ config BOARD
 	default "qemu_x86"
 
 endif # BOARD_QEMU_X86
+
+if BOARD_QEMU_X86_IAMCU
+
+config BUILD_OUTPUT_BIN
+        default n
+
+config BOARD
+	default "qemu_x86_iamcu"
+
+# Enabled here because IAMCU targets run on some Quark CPUs that are immune to
+# these attacks, and the compiler doesn't even recognize instructions like
+# 'lfence'
+# We set this here because the QEMU targets derive from the generic IA32 SOC
+# definition and not a specific CPU; this is normally set at the SOC level.
+config X86_NO_SPECULATIVE_VULNERABILITIES
```

Comment:
based on discussion on slack, I think rather than introducing this here, we should add:

`select X86_NO_SPECULATIVE_VULNERABILITIES` in arch/x86/Kconfig for `config CPU_MINUTEIA`.  (Assuming that's right for all CPU_MINUTEIA class SoC.

That will get what we want for qemu_x86_iamcu rather than introducing this Kconfig foo here.